public/scss/components/PopUpVoucherCoupon.module.scss

```
@@ -455,10 +536,174 @@
   object-fit: contain;
 }
 
-.voucherLoading
+.voucherShowMoreContainer
+{
+  @include flex(row, center, center);
+}
+
+.voucherShowMoreButton
+{
+  @extend .btn;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+  color: $color_blue;
+
+  &:hover,
+  &:active,
+  &:focus
+  {
+    background-color: transparent;
+    color: $color_blue;
+  }
+}
+
+.voucherDetailPopUpContainer
+{
+  @extend .popup_overlay;
+  @include flex(row, center, center);
+  @include fixedWidth(100%);
+  @include fixedHeight(100%);
```

```suggestion
  @include fixedSize(100%);
```

pake mixin ini saja ngab